io.github.franjegueje.deckico.yml

```diff
@@ -0,0 +1,30 @@
+app-id: io.github.franjegueje.deckico
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: deckico-gui
+finish-args:
+  - --socket=x11
+  - --filesystem=~/.config/autostart
+  - --filesystem=~/.local/share/Steam/steamapps/compatdata
+  - --filesystem=~/.local/share/Steam/steamapps/shadercache
+  - --filesystem=~/.local/share/Steam/userdata:ro
+  - --filesystem=/run/media
+modules:
+  - name: deckico
+    buildsystem: simple
+    build-commands:
+      - install -D deckico-gui /app/bin/deckico-gui
+      - install -D deckico-cli /app/bin/deckico-cli
+      - install -D yad /app/bin/yad
+      - mkdir -p /app/usr/share/icons/deckico
+      - tar -xvzf images.tar.gz -C /app/usr/share/icons/deckico/.
+    sources:
+      - type: file
+        path: deckico-gui
+      - type: file
+        path: deckico-cli
```

Comment:
these are part of "upstream" therefor they don't belong here, as I previously said.